src/audio/copier.c

```diff
@@ -410,6 +409,7 @@ static struct comp_dev *copier_new(const struct comp_driver *drv,
 		};
 	}
 
+	dev->pipeline = ipc_pipe->pipeline;
```

Comment:
this line will be easier to read if you set this right after line 362

Reply:
sure, thanks